src/theme/index.ts

```diff
@@ -0,0 +1,30 @@
+export const theme = {
+  colors: {
+    bg: '#130F1C',
+    txt: '#FFF',
+    gray: {
+      900: '#1F1B29',
+      800: '#29292E',
+      700: '#2E2A36',
+      600: '#3C3A41',
+      500: '#4F4D54',
+    },
+    purple: {
+      800: '#651FFF',
+      700: '#8046FF',
+      600: '#9D72FD',
+    },
+    red: {
+      800: '#BB3F3F',
+    },
+    cyan: {
+      900: '#0F9B8E',
+      800: '#3DA59B',
+      700: '#4EBAB0',
+    },
+    green: {
+      900: '#04D361',
+      800: '#50E793',
+    },
+  },
+};
```

Comment:
@cassiosilva93 I think we can improve this theme by adding attributes like defaultPadding, defaultMargin and etc. This will improve the consistency of the project. We can discuss this later. I'll talk to some friends that uses Design systems.